src/main/scala/com/fortysevendeg/translatebubble/ui/commons/ListLayout.scala

```diff
@@ -33,6 +34,15 @@ trait ListLayout
 
   var placeholderContent = slot[LinearLayout]
 
+  def layoutManager(implicit contextWrapper: ContextWrapper) = if (landscapeTablet) {
+    new GridLayoutManager(contextWrapper.application, 3)
+  } else if (tablet) {
+    new GridLayoutManager(contextWrapper.application, 2)
+  } else {
+    new LinearLayoutManager(contextWrapper.application)
+  }
```

Comment:
Make sense to improve the `landscape` mode in all devices?
´´´scala
(tablet, landscape) match {
  case (true, true) => new GridLayoutManager(contextWrapper.application, 3)
  case (true, false) => new GridLayoutManager(contextWrapper.application, 2)
  case (false, true) => new GridLayoutManager(contextWrapper.application, 2)
  case _ => new LinearLayoutManager(contextWrapper.application)
}
```